javascript-and-json/main.js

```diff
@@ -0,0 +1,30 @@
+var bookList = [
+  {
+    isbn: '12345-123',
+    title: 'Harry Potter',
+    author: 'J.K. Rowling'
+  },
+  {
+    isbn: '12345-124',
+    title: 'Star Wars',
+    author: 'George Lucas'
+  },
+  {
+    isbn: '12345=125',
+    title: 'Title',
+    author: 'Author'
+  }
+];
+console.log('value of bookList: ', bookList);
+console.log('value of typeof bookList: ', typeof bookList);
+
+JSON.stringify(bookList);
```

Comment:
`JSON.stringify` does not change the original value, you need to save this in a variable to see any actual change.